.github/workflows/publish-cloud-registry.yml

```diff
@@ -3,7 +3,7 @@ name: Publish to Google Cloud Registry
 on:
   push:
     tags:
-      - "@commercetools-frontend/mc-http-server@*"
+      - "*"
```

Comment:
PS: let's see if this helps triggering the workflow. The job itself has its own condition anyway.